EssentialCSharp.Web/Views/Home/Home.cshtml

```diff
@@ -28,7 +28,9 @@
     <div class="row gx-5">
         <div class="col-12 col-md-6">
             <div class="d-flex align-items-center justify-content-center h-100">
-                <img src="/images/book.png" class="img-fluid" style="height: 450px" alt="Essential C Sharp Book">
+                <a href="https://a.co/d/95wxhTg" target="_blank">
```

Comment:
This looks great! Do you mind also applying this to the image on the /about page?

Reply:
Done! Didn't realize there was the book cover in the /about page. Thanks :)